addons/binding/org.openhab.binding.bigassfan/src/main/java/org/openhab/binding/bigassfan/handler/BigAssFanHandler.java

```diff
@@ -429,7 +502,7 @@ private void updateChannel(String channelName, State state) {
     }
 
     private String thingID() {
-        return thing.getUID().getId();
+        return thing.getUID().toString();
```

Comment:
This does not look well, are you sure you want to do this? Where do you use it for.